components/camel-script/src/main/java/org/apache/camel/builder/script/ScriptBuilder.java

```diff
@@ -32,6 +32,9 @@
 import javax.script.ScriptEngineManager;
 import javax.script.ScriptException;
 
+import jdk.nashorn.api.scripting.NashornScriptEngine;
+import jdk.nashorn.api.scripting.ScriptObjectMirror;
```

Comment:
Not sure if we can import this package in Camel code as it should only use the public api from JDK, eg that is typically java. For example not sure if OpenJDK or IBM JDKs have this jdk.nashorn package.

Can you research some more on the internet about this